### PR TITLE
Slack messaging

### PR DESCRIPTION
**Pull request by billbitt**
added `winston-slack-webhook` npm package and configured to send spee.ch logs to slack.
- error logs are being sent to #slack-errors
- info logs are being sent to #slack-logs
both are configurable in the `/config/production.json` file

### CODE REVIEW

**Comment by kauffj** [comment]
Is this WIP or actually needs review?

File: helpers/errorHandlers.js [hunk 1]
```diff
@@ -1,32 +1,38 @@
 const logger = require('winston');
 const { postToStats } = require('../controllers/statsController.js');
 
+function useObjectPropertiesIfNoKeys (err) {
+  if (Object.keys(err).length === 0) {
+    let newErrorObject = {};
+    Object.getOwnPropertyNames(err).forEach((key) => {
```
Comment (kauffj) [inline helpers/errorHandlers.js]: When is there a non-enumerable property being passed that this is required?

File: package.json [hunk 2]
```diff
@@ -39,7 +39,8 @@
     "socket.io": "^2.0.1",
     "socketio-file-upload": "^0.6.0",
     "universal-analytics": "^0.4.13",
-    "winston": "^2.3.1"
+    "winston": "^2.3.1",
+    "winston-slack-webhook": "billbitt/winston-slack-webhook"
```
Comment (kauffj) [inline package.json]: should this be billbitt?

**Comment by billbitt** [comment]
@kauffj review if you have any input on it.  It's not central to key spee.ch functionality, but necessary for monitoring status.

File: helpers/errorHandlers.js (hunk 1 above)
Comment (billbitt) [reply helpers/errorHandlers.js]: for errors created by node (e.g. `new Error('error message')`) the stack and message are non-enumerable

File: package.json (hunk 2 above)
Comment (billbitt) [reply package.json]: I made my own fork to fix an issue with the package, and submitted a PR to the main package.  The PR was accepted, so will update this to point to main package in next update.

**Pull request merged**